deploy/docker/entrypoint.sh

```diff
@@ -383,6 +383,7 @@ init_postgres || runEmbeddedPostgres=0
 }
 
 init_loading_pages(){
+  sed -i '/\[::\]:80 default_server;/d' /etc/nginx/sites-enabled/default
```

Comment:
Let's add a commend describing what this is please?

```suggestion
  # The default NGINX configuration includes an IPv6 listen directive. But not all
  # servers support it, and we don't need it. So we remove it here before starting
  # NGINX.
  sed -i '/\[::\]:80 default_server;/d' /etc/nginx/sites-enabled/default
```